requirements.txt

```diff
@@ -23,4 +23,5 @@ git+https://github.com/armmbed/manifest-tool.git@v1.4.6
 icetea>=1.2.1,<1.3
 pycryptodome>=3.7.2,<=3.7.3
 pyusb>=1.0.0,<2.0.0
+hidapi>=0.7.99,<0.8.0
```

Comment:
@c1728p9, @jamesbeyond I'd like to clarify the licensing of this module. Based on its [LICENSE file](https://github.com/trezor/cython-hidapi/blob/master/LICENSE.txt) it can be used under one of 3 licenses. Should I specify, which one we want to use? If so, how do I do that?

Reply:
@0xc0170 Might be a question for you as well

Reply:
I need more context here, one of 3 licenses? Does this module hidapi have dual license or?

Reply:
Yes 3
> cython-hidapi may be used by one of three licenses as outlined in LICENSE.txt

Reply:
Got it . This is missing in our license file (capturing python dependencies there!) , can you fix that ? IT would be added there as well - hidapi under BSD license

Reply:
Created #10123